### DISCUSSION TITLE
How to identify current activated theme from module?

### DISCUSSION THREAD

**Discussion by kudryavtsevda** [OrchardCMS/OrchardCore#11545] [Q&A]
I have a module which is working from admin dashboard and I need to identify the name of current activated admin/client theme and other params related to that theme from my custom module. 
How can I do it from my custom module controller or from any other places inside my custom module?

**Comment by ns8482e** [comment]
Why do you need to know theme from module, module should  not be dependent on theme

**Reply by kudryavtsevda** [reply]
> Why do you need to know theme from module, module should not be dependent on theme

I am just curious. Let's say I want to display in my module the list of current activated modules and themes from admin dashboard

**Reply by ns8482e** [reply]
I guess there is no easy way,  but you can check the code in Themes module

**Comment by hishamco** [comment]
>  I need to identify the name of current activated admin/client theme and other

Use can use `IAdminThemeService.GetAdminThemeNameAsync()` for the selected admin theme and `ISiteThemeService.GetCurrentThemeNameAsync()` for the selected site theme
